app/views/articles/_actions.html.erb

```diff
@@ -9,11 +9,13 @@
     <button id="reaction-butt-readinglist" class="article-reaction-butt readinglist-reaction-button" data-category="readinglist" alt="unicorn-emoji" title="reading list">
       <img alt="reading list" src="<%= asset_path("emoji/emoji-one-bookmark.png") %>" /><span class="reaction-number" id="reaction-number-readinglist"></span>
     </button>
-    <a class="article-actions-tweet-button"
-      id="article-actions-tweet-button"
+
+    <% display_name = @article.user.twitter_username ? "@" + @article.user.twitter_username : @article.user.name %>
+    <a
+      class="article-actions-tweet-button"
       rel="noopener"
       target='_blank'
-      href='https://twitter.com/intent/tweet?text="<%= u @article.title.strip %>" by <%= @article.user.twitter_username ? "@" + @article.user.twitter_username : @article.user.name %> %23DEVcommunity https://dev.to<%= @article.path %>'>
+      href='https://twitter.com/intent/tweet?text="<%= u @article.title.strip %>" by <%= display_name %> %23DEVcommunity <%= article_url(@article) %>'>
```

Comment:
unrelated, but I noticed this when opening the file to understand the HTML. Let me know if I should remove it

We should "parameterize" the `#DEVcommunity` for twitter shares as well but it's outside the scope